cached_repository.go

```diff
@@ -16,6 +16,9 @@ type CachedRepository interface {
 	// If successful, then the value is also updated/cached in the configured caches and/or evicted
 	// from the caches that requested eviction on write operations
 	Update(ctx context.Context, value interface{}) error
+	// If successful, then the value is also updated/cached in the configured caches and/or evicted
```

Comment:
Can you update the comment to indicate that the fully updated object will be retrieved from the datastore and cached please?

Reply:
Also, point out that there is a side-effect that the `value` is/can be used as an input/output, is that right?

The actual value changes after the operation in Gorm (assuming people use the Gorm backend). Please correct me if I'm wrong :)

Reply:
yeah, received object is complemented with information that is saved in data storage, and `value` received is changed after gorm operation

Reply:
done